lib/rumbrl/ext/hash.rb

```diff
@@ -0,0 +1,8 @@
+# Extensions for native Hash to allow for log ingestion
+module HashExtensions
+  def to_log
+    to_hash
+  end
+end
+
+::Hash.include(HashExtensions)
```

Comment:
I still dont get what it gets us though? what else would respond to this except for Hashes?